tooltip.lua

```diff
@@ -9,10 +9,21 @@ function ns.dataobj.OnTooltipShow(tip)
 	if num then
 		for i=1,num do
 			local specID, spec = GetSpecializationInfo(i)
-			tip:AddLine(" ")
-			tip:AddLine(spec)
-			for j,stat in pairs(ns.priorities[specID]) do
-				tip:AddLine(stat, 1,1,1)
+			local priorities = ns.priorities[specID]
+			if type(priorities[1]) ~= "table" then
+				priorities = {priorities}
```

Comment:
Fair enough. One or two extra tables when you mouseover an LDB entry shouldn't matter, but I may as well do it the cleaner way.